packages/react/src/editor/styles.css

```diff
@@ -0,0 +1,414 @@
+    padding: 8px;
+}
+
+/* Mantine TextInput component base styles */
+.mantine-TextInput-input {
+    background-color: var(--bn-color-menu-background);
+    border: solid var(--bn-color-border) 1px;
+    border-radius: 4px;
+    color: var(--bn-color-menu-text);
+    height: 32px;
+}
+
+
+/* Editor body styling */
+
+/* Editor styling */
+.bn-editor {
+    background-color: var(--bn-color-editor-background);
+    border-radius: var(--bn-border-radius-large);
+    color: var(--bn-color-editor-text);
+    font-family: var(--bn-font-family);
+}
+
+/* Indent line styling */
+.bn-block-group
+.bn-block-group
+.bn-block-outer:not([data-prev-depth-changed])::before {
+    border-left: 1px solid var(--bn-color-side-menu)
+}
+
+/* Placeholder styling */
+.bn-is-empty .bn-inline-content:before, .bn-is-filter .bn-inline-content:before {
+    color: var(--bn-color-side-menu);
+}
+
+/* Highlight color styling */
+[data-text-color="gray"] {
+    color: var(--bn-color-highlight-text-gray);
+}
+[data-text-color="brown"] {
+    color: var(--bn-color-highlight-text-brown);
+}
+[data-text-color="red"] {
+    color: var(--bn-color-highlight-text-red);
+}
+[data-text-color="orange"] {
+    color: var(--bn-color-highlight-text-orange);
+}
+[data-text-color="yellow"] {
+    color: var(--bn-color-highlight-text-yellow);
+}
+[data-text-color="green"] {
+    color: var(--bn-color-highlight-text-green);
+}
+[data-text-color="blue"] {
+    color: var(--bn-color-highlight-text-blue);
+}
+[data-text-color="purple"] {
+    color: var(--bn-color-highlight-text-purple);
+}
+[data-text-color="pink"] {
+    color: var(--bn-color-highlight-text-pink);
+}
+[data-background-color="gray"] {
+    background-color: var(--bn-color-highlight-background-gray);
+}
+[data-background-color="brown"] {
+    background-color: var(--bn-color-highlight-background-brown);
+}
+[data-background-color="red"] {
+    background-color: var(--bn-color-highlight-background-red);
+}
+[data-background-color="orange"] {
+    background-color: var(--bn-color-highlight-background-orange);
+}
+[data-background-color="yellow"] {
+    background-color: var(--bn-color-highlight-background-yellow);
+}
+[data-background-color="green"] {
+    background-color: var(--bn-color-highlight-background-green);
+}
+[data-background-color="blue"] {
+    background-color: var(--bn-color-highlight-background-blue);
+}
+[data-background-color="purple"] {
+    background-color: var(--bn-color-highlight-background-purple);
+}
+[data-background-color="pink"] {
+    background-color: var(--bn-color-highlight-background-pink);
+}
+
+/* UI element styling */
+
+/* Toolbar styling */
+.bn-toolbar {
+    background-color: var(--bn-color-menu-background);
+    border: var(--bn-border);
+    border-radius: var(--bn-border-radius-medium);
+    box-shadow: var(--bn-shadow-medium);
+    flex-wrap: nowrap;
+    gap: 2px;
+    padding: 2px;
+    width: fit-content;
+
+    & .mantine-Button-root, .mantine-ActionIcon-root {
```

Comment:
is this nesting valid in css or only in scss?

Reply:
I think it's a fairly recent change, but it's valid for vanilla CSS: https://developer.mozilla.org/en-US/docs/Web/CSS/Nesting_selector

Though I noticed it's causing issues in docker, so will probably have to change it either way

Reply:
interesting, didn't know. But support is too limited for now indeed